cs.html

```diff
@@ -1710,60 +1931,61 @@ <h2>Event summary</h2>
-    ignored in this example).</p>
-    <pre class="example highlight">
-    const quic = getQuicTransport();
-    const datagrams = getDatagramsToSend();
-    datagrams.forEach((datagram) => {
-      await quic.readyToSendDatagram();
-      quic.sendDatagram(datagram);
-    });
-    </pre>
-  </section>
-  <section class="informative" id="datagramexample2*">
-    <h3>Sending QUIC datagrams at a fixed rate</h3>
-    <p>Sending QUIC datagrams at a fixed rate regardless if the transport is ready to
-    send can be achieved by simply using <code>sendDatagram</code> and not using
-    the <code>readyToSendDatagram</code> method. More complex scenarios can utilize
-    the <code>readyToSendDatagram</code> method.</p>
-    <pre class="example highlight">
-    // Sends datagrams every 100 ms.
-    const quic = getQuicTransport();
-    setInterval(() => {
-      quic.sendDatagram(createDatagram());
-    }, 100);
-    </pre>
-  </section>
-  <section class="informative" id="datagramexample3*">
-      <h3>Receiving QUIC datagrams</h3>
-      <p>Receiving QUIC datagrams can be achieved by simply listening to the
+let transport = getTransport();
+let messages = getMessages();
+for (msg in messages) {
+  transport.createSendStream({disableRetransmissions: true}).write({data: msg, finished: true});
+}</pre>
+    </section>
+
+    <section class="informative" id="datagramexample1*">
+      <h3>Sending a buffer of datagrams</h3>
+      <p>Sending a buffer of datagrams can be achieved by using the
+      <code>sendDatagram</code> and <code>readyToSendDatagram</code> methods. In
+      the following example datagrams are only sent if the
+      <code>DatagramTransport</code> is ready to send, however the sending is not
+      blocked on the ACK promise returned from <code>sendDatagram</code> (these are
+      ignored in this example).</p>
+      <pre class="example highlight">
+const transport = getTransport();
+const datagrams = getDatagramsToSend();
+datagrams.forEach((datagram) => {
+  await transport.readyToSendDatagram();
+  transport.sendDatagram(datagram);
+});</pre>
+    </section>
+
+    <section class="informative" id="datagramexample2*">
+      <h3>Sending datagrams at a fixed rate</h3>
+      <p>Sending datagrams at a fixed rate regardless if the transport is ready to
+      send can be achieved by simply using <code>sendDatagram</code> and not using
+      the <code>readyToSendDatagram</code> method. More complex scenarios can utilize
+      the <code>readyToSendDatagram</code> method.</p>
+      <pre class="example highlight">
+// Sends datagrams every 100 ms.
+const transport = getTransport();
+setInterval(() => {
+  transport.sendDatagram(createDatagram());
+}, 100);</pre>
+    </section>
+
+    <section class="informative" id="datagramexample3*">
+      <h3>Receiving datagrams</h3>
+      <p>Receiving datagrams can be achieved by simply listening to the
       <code>datagramsreceived</code>, remembering to check for null values
```

Comment:
datagramsreceived is not defined in the current text.

Reply:
Removed